apps/spruce/src/pages/projectSettings/tabs/GithubAppSettingsTab/getFormSchema.tsx

```diff
@@ -113,12 +121,14 @@ export const getFormSchema = ({
           "ui:data-cy": "github-app-id-input",
           "ui:disabled": isAppDefined,
           "ui:elementWrapperCSS": appFieldCss,
+          ...placeholderIf(repoData?.appCredentials?.githubAppAuth?.appId),
         },
         privateKey: {
           "ui:data-cy": "github-private-key-input",
           "ui:disabled": isAppDefined,
           "ui:elementWrapperCSS": appFieldCss,
           "ui:widget": "textarea",
+          ...placeholderIf(repoData?.appCredentials?.githubAppAuth?.privateKey),
         },
       },
       actions: {
```

Comment:
It's confusing that the yellow warning banner shows when the project is defaulting to repo; it seems to imply that the GitHub app is not defined

Reply:
I think the banner is still accurate here because they won't be able to edit the permissions when it's defaulting to the repo (it's true that it's not defined on the project because it's using the repo instead)